src/main/java/io/ticktok/Listener/TickListener.java

```diff
@@ -13,6 +17,7 @@ public static void listen(Channel channel, Runnable runnable) throws TicktokExce
             com.rabbitmq.client.Channel tickChannel = listen(channel);
             Consumer consumer = consume(runnable, tickChannel);
             tickChannel.basicConsume(channel.getQueue(), true, consumer);
+            log.debug(MessageFormat.format("now listening on queue : {0}", channel.getQueue()));
```

Comment:
I think you can simplify and do: log.debug("now listening on queue : {}", channel.getQueue()));
And it will also wont parse if it's not in debug mode

Reply:
can't fix it, when using MessageFormat you have to give # to a param link - `{0}`
otherwise it will throw illegal argument exception.
other ideas?

Reply:
you just give {} and it will go by order

Reply:
tried that, and it fails on `illegal argument exception`.
you want to send me a snippet that works for you?

Reply:
fixed